lib/render/mayaToHydra/renderOverrideUtils.h

```diff
@@ -0,0 +1,145 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef __MTOH_VIEW_OVERRIDE_UTILS_H__
+#define __MTOH_VIEW_OVERRIDE_UTILS_H__
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+class HdMayaSceneRender : public MHWRender::MSceneRender {
+public:
+    explicit HdMayaSceneRender(const MString& name, bool drawSelectionOverlay)
+        : MHWRender::MSceneRender(name),
+          _drawSelectionOverlay(drawSelectionOverlay) {}
+
+    MUint64 getObjectTypeExclusions() override {
+        return _drawSelectionOverlay
+                   ? MHWRender::MSceneRender::getObjectTypeExclusions()
+                   : ~(MHWRender::MFrameContext::kExcludeSelectHandles |
+                       MHWRender::MFrameContext::kExcludeCameras |
+                       MHWRender::MFrameContext::kExcludeCVs |
+                       MHWRender::MFrameContext::kExcludeDimensions |
+                       MHWRender::MFrameContext::kExcludeLights |
+                       MHWRender::MFrameContext::kExcludeLocators |
+                       MHWRender::MFrameContext::kExcludeGrid);
+    }
+
+    MSceneFilterOption renderFilterOverride() override {
+        return _drawSelectionOverlay
+                   ? kRenderUIItems
+                   : MHWRender::MSceneRender::renderFilterOverride();
+    }
+
+    MHWRender::MClearOperation& clearOperation() override {
+        auto* renderer = MHWRender::MRenderer::theRenderer();
+        const auto gradient = renderer->useGradient();
+        const auto color1 = renderer->clearColor();
+        const auto color2 = renderer->clearColor2();
+
+        float c1[4] = {color1[0], color1[1], color1[2], 1.0f};
+        float c2[4] = {color2[0], color2[1], color2[2], 1.0f};
+
+        mClearOperation.setClearColor(c1);
+        mClearOperation.setClearColor2(c2);
+        mClearOperation.setClearGradient(gradient);
+        return mClearOperation;
+    }
+
+    bool _drawSelectionOverlay = false;
+};
+
+class HdMayaManipulatorRender : public MHWRender::MSceneRender {
+public:
+    explicit HdMayaManipulatorRender(const MString& name)
+        : MHWRender::MSceneRender(name) {}
+
+    MUint64 getObjectTypeExclusions() override {
+        return ~MHWRender::MFrameContext::kExcludeManipulators;
+    }
+
+    MHWRender::MClearOperation& clearOperation() override {
+        mClearOperation.setMask(MHWRender::MClearOperation::kClearNone);
+        return mClearOperation;
+    }
+};
+
+class HdMayaRender : public MHWRender::MUserRenderOperation {
+public:
+    HdMayaRender(const MString& name, MtohRenderOverride* override)
+        : MHWRender::MUserRenderOperation(name), _override(override) {}
+
+    MStatus execute(const MHWRender::MDrawContext& drawContext) override {
+        return _override->Render(drawContext);
+    }
+
+    bool hasUIDrawables() const override { return false; }
+
+    bool requiresLightData() const override { return false; }
```

Comment:
There is no need to override these two functions as the behavior is the same as the default.

Reply:
Done:
https://github.com/Autodesk/maya-usd/pull/231/commits/5999ef0aa0b2be2f805a2b082e6167458907400e